.github/workflows/create-branch-on-tag.yml

```diff
@@ -0,0 +1,63 @@
+name: Create Branch on Tag Push
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'pantheon-systems/search_api_pantheon' }}
+    env:
+      BRANCH: ${{ github.ref_name }}
+      WORKSPACE: ${{ github.workspace }}
+      DRUPAL_ORG_REMOTE: ${{ secrets.DRUPAL_ORG_REMOTE }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SSH_KEY }}
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+          if_key_exists: ignore
+      - name: Extract tag name
+        id: tag_name
+        run: |
+          FULL_TAG=${GITHUB_REF#refs/tags/}
+          MAJOR_MINOR="$(echo $FULL_TAG | cut -d '.' -f 1,2).x"
+          MAJOR_ONLY="$(echo $FULL_TAG | cut -d '.' -f 1).x"
+          echo "BRANCH_NAME=$MAJOR_MINOR" >> $GITHUB_OUTPUT
+          echo "MAJOR_BRANCH_NAME=$MAJOR_ONLY" >> $GITHUB_OUTPUT
```

Comment:
These lines, as well as the ` >> $GITHUB_OUTPUT` lines on 42 and 45 are causing this warning:
`Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/`

I don't know what "soon" means given that the changelog linked is from 2022 but probably best to change the behavior anyway before the workflow breaks from under us.

Reply:
That was fixed in the next commit, as you can see https://github.com/rwagner00/search_api_pantheon/actions/runs/7282092388 there are no more deprecation notices